app/build.gradle

```diff
@@ -142,6 +142,7 @@ dependencies {
     implementation "com.android.support:support-annotations:$rootProject.supportLibraryVersion"
     implementation 'com.android.support.constraint:constraint-layout:1.1.2'
     implementation "com.android.support:multidex:$rootProject.multidexVersion"
+    implementation 'com.google.android.gms:play-services-auth:15.0.1'
```

Comment:
Is it a non free dependency ? If so, we should only enable it with a flag.
--build-with-non-free-components

Reply:
@jaindiv26 I'm concerned about all those imports. Can't they be made conditional @Karljoones @ross-holloway94 ?